themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -122,7 +122,7 @@
                                 <#nested "show-username">
                                 <div id="kc-username" class="${properties.kcFormGroupClass!}">
                                     <#nested "loginTotpCodeInputTitle">
-                                    <label id="kc-attempted-username">${auth.attemptedUsername}</label>
+                                    <label id="kc-attempted-username">${msg("x509ModalHeader")}</label>
```

Comment:
@JonSzeto821 are we sure this doesn't affect any other flows?

Reply:
I've updated this to be handled by a conditional depending on what screen is being displayed.